settings.sample.py

```diff
@@ -84,13 +91,17 @@ def env(key, default, cast=str):
 # prevent really noisy reviews from slowing down github.
 SUMMARY_THRESHOLD = env('LINTREVIEW_SUMMARY_THRESHOLD', 50, int)
 
-# This config file contains default settings for .lintrc
-# LINTRC_DEFAULTS = './lintrc_defaults.ini'
+# Status Configuration
+######################
+
+# Uncomment this option to enable adding an issue comment
+# whenever a pull request passes all checks.
+# OK_COMMENT = env('LINTREVIEW_OK_COMMENT',
+#                 ':+1: No lint errors found.')
 
-OK_COMMENT = env('LINTREVIEW_OK_COMMENT',
-                 ':+1: No lint errors found.')
+# Set to True to apply a label when updating build status.
+# Pull requests that fail will have the label removed.
+# ADD_OK_LABEL = env('LINTREVIEW_ADD_OK_LABEL', False, bool)
```

Comment:
What about removing `ADD_OK_LABEL`, and only add the label if `OK_LABEL` is set?